.github/workflows/build_and_test.yml

```diff
@@ -253,7 +253,14 @@ jobs:
       build-tag: ${{ needs.tag.outputs.build-tag }}
       build-type: ${{ matrix.build-type }}
       # Run tests on all Postgres versions in release builds and only on the latest version in debug builds
-      pg-versions: ${{ matrix.build-type == 'release' && '["v14", "v15", "v16", "v17"]' || '["v17"]' }}
+      # run without LFC on v17 release only
+      test-cfg: |
+        ${{ matrix.build-type == 'release' && '[{"pg_version":"v14", "lfc_state": "without-lfc"},
+                                                {"pg_version":"v15", "lfc_state": "without-lfc"},
+                                                {"pg_version":"v16", "lfc_state": "without-lfc"},
+                                                {"pg_version":"v17", "lfc_state": "without-lfc"},
+                                                {"pg_version":"v17", "lfc_state": "with-lfc"}]'
+                                           || '[{"pg_version":"v17", "lfc_state": "without-lfc"}]' }}
```

Comment:
Would an `lfc` boolean be better?

Reply:
At the vary least `use_lfc` would be good to match the `USE_LFC` python variable.

Reply:
Using the boolean variable `lfc` is quite natural. But, on the other hand, the test, in the list would look like this:
```
Build and Test / build-and-test-locally (arm64, release) / regress-tests (v16, true)
```
So `true` or `false` in this case may confuse.
The name `use_lfc` was also discussed, but we refused it because it assumes a boolean, which is confusing.